### ISSUE TITLE
md-tab - label not visible if it contains a space and browser is resized

### ISSUE THREAD

**Issue by rhino5oh** [angular/material2#5351]
#### Bug, feature request, or proposal:
Bug.  If you have an md-tab, with a label that contains a space, the wording after the space is no longer visible if the browser window size is reduced.  For example, if your label says "Medical History", it displays fine when the browser window is large.  However, if you make the browser window small enough, the second word is cut off and all you see if "Medical" in the tab. See attached screenshot.

#### What is the expected behavior?
The tab should still show the full text, even if the browser window is resized

#### What is the current behavior?
The tab label gets cut off

#### What are the steps to reproduce?
create and md-tab with a multi-word label (with a space in between the words), and shrink your browser window down

Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
Plunker template: https://goo.gl/DlHd6U

#### What is the use-case or motivation for changing an existing behavior?
its a bad user experience the way it currently is

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Discoved while using Angular 4.1.2 and Material 2.0.0-beta.7

#### Is there anything else we should know?
negatory
![capture](https://user-images.githubusercontent.com/4404630/27511682-8c9cf40c-58f8-11e7-9dd3-f2b63b6a8515.JPG)

**Comment by julianobrasil** [comment]
Yeah... but according to Material Design specs, there's a limit to show the full text and the tabs shold have the same width. In this case, it seems it should have broken the line and decreased the font size.

![image](https://user-images.githubusercontent.com/25405143/27511757-b8de7ac2-5902-11e7-9166-3293f9e13ca3.png)

**Comment by rhino5oh** [comment]
I agree @julianobrasil, thanks for your response.  That seems like a perfectly acceptable way to handle text over the limit.  Either way, it should never completely cut off what should be there without any indication that there is more information in the label

**Comment by soarfalc** [comment]
is possible to fix this problem ? :)

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
